### PR TITLE
Add changelog for v4.1.0

### PR DESCRIPTION
**Pull request by jsafrane**
**What type of PR is this?**
/kind documentation

Add changelog for v4.1.0. When at it, I marked CSI migration as a GA feature, it was GA already for some CSI drivers, now it's GA for all cloud based ones.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The following table reflects the head of this branch.
 
 | Feature       | Status  | Default | Description                                                                                   |
 | ------------- | ------- | ------- | --------------------------------------------------------------------------------------------- |
-| CSIMigration*     | Beta    | On      | [Migrating in-tree volume plugins to CSI](https://kubernetes.io/docs/concepts/storage/volumes/#csi-migration). |
+| CSIMigration*     | GA    | On      | [Migrating in-tree volume plugins to CSI](https://kubernetes.io/docs/concepts/storage/volumes/#csi-migration). |
```
Comment (humblec) [inline README.md:33]: @jsafrane did we loose indentation here?

File: CHANGELOG/CHANGELOG-4.1.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
+# Release notes for v4.1.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v4.0.0
+
+### Uncategorized
+
+- Kube client dependencies are updated to v1.26.0.
```
Comment (xing-yang) [inline CHANGELOG/CHANGELOG-4.1.md:9]: Can you add 
`([#PR number](https://github.com/kubernetes-csi/external-attacher/pull/<xxxx>), [@PR author git id](https://github.com/git id))`?

File: README.md (hunk 1 above)
Comment (jsafrane) [reply README.md]: Fixed

File: CHANGELOG/CHANGELOG-4.1.md (hunk 2 above)
Comment (jsafrane) [reply CHANGELOG/CHANGELOG-4.1.md]: Fixed

**Comment by humblec** [comment]
Thanks.!  lgtm

**Pull request merged**